nvm.el

```diff
@@ -4,10 +4,10 @@
 
 ;; Author: Johan Andersson <johan.rejeep@gmail.com>
 ;; Maintainer: Johan Andersson <johan.rejeep@gmail.com>
-;; Version: 0.1.0
+;; Version: 0.1.1
```

Comment:
Can you please remove this. I usually do a separate commit with this and also release a tag for it.